src/components/Recipe.test.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import {
+  render,
+  screen,
+  waitForElementToBeRemoved,
+} from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import Recipe from "./Recipe.js";
+import { Route, MemoryRouter } from "react-router-dom";
+import api from "../data/DataAPI.js";
+
+const recipeResponse = {
+  id: 1,
+  name: "Empanadas",
+  description: "Carne envuelta en masa",
+  ingredients: [
+    {
+      name: "Carne",
+    },
+    {
+      name: "Masa",
+    },
+  ],
+};
+
+const mockFetchRecipe = jest.spyOn(api, "fetchRecipe");
+mockFetchRecipe.mockReturnValue(Promise.resolve(recipeResponse));
```

Comment:
much better but you can also check that the method has been called with the proper params